draft-add-requirements.md

```diff
@@ -104,92 +108,137 @@ the IETF may publish, such as DNS-over-QUIC {{?I-D.ietf-dprive-dnsoquic}}.
 
 Do53: Unencrypted DNS over UDP port 53, or TCP port 53 {{?RFC1035}}.
 
-Equivalent: See {{equivalence}}.
+Designated: See {{designation}}.
+
+Designator: The network or resolver that issued the designation.
 
 
-# Use case description
+# Overall use case
```

Comment:
"Overall" sounds a bit awkward in the title here. What was wrong with the old section title?

Reply:
I think I did this because I was struggling with the idea of a single use case that has multiple sub-cases. "Overall" was an attempt to convey this was a description of the over-arching concept, with the nuances described later. But I'm happy to revert to "Use case description" if that works better.

Reply:
use case seems fine to me.

Reply:
Now reverted back.